com.unity.netcode.gameobjects/Runtime/Messaging/Messages/NetworkVariableDeltaMessage.cs

```diff
@@ -69,7 +69,12 @@ public void Serialize(FastBufferWriter writer)
                         var tmpWriter = new FastBufferWriter(MessagingSystem.NON_FRAGMENTED_MESSAGE_MAX_SIZE, Allocator.Temp, short.MaxValue);
                         NetworkBehaviour.NetworkVariableFields[k].WriteDelta(tmpWriter);
 
-                        writer.WriteValueSafe((ushort)tmpWriter.Length);
+                        if (!writer.TryBeginWrite(FastBufferWriter.GetWriteSize<ushort>() + tmpWriter.Length))
```

Comment:
While this does not fix the issue with NetworkLists that exceed the MessagingSystem.NON_FRAGMENTED_MESSAGE_MAX_SIZE, I put this in place as it was missed in the back port.
This will not actually fix the issue until [MTT-2434](https://jira.unity3d.com/browse/MTT-2434) is resolved but should handle this particular issue within NetworkVariableDeltaMessage.